src/server/src/database/databaseClient.js

```diff
@@ -0,0 +1,44 @@
+export class DatabaseClient {
+  #client;
```

Comment:
for my own knowledge what's this syntax?

Reply:
private member variables and methods.